pkg/hubble/parser/parser_test.go

```diff
@@ -17,18 +17,17 @@
 package parser
 
 import (
-	"bytes"
-	"encoding/gob"
-	"io/ioutil"
-	"testing"
-
 	flowpb "github.com/cilium/cilium/api/v1/flow"
 	v1 "github.com/cilium/cilium/pkg/hubble/api/v1"
+	observerTypes "github.com/cilium/cilium/pkg/hubble/observer/types"
 	"github.com/cilium/cilium/pkg/hubble/parser/errors"
 	"github.com/cilium/cilium/pkg/hubble/testutils"
 	"github.com/cilium/cilium/pkg/monitor"
 	"github.com/cilium/cilium/pkg/monitor/api"
 	"github.com/cilium/cilium/pkg/proxy/accesslog"
+	"io/ioutil"
```

Comment:
nit: you probably didn't mean to group stdlib imports with the others.

Reply:
Yes.